std/experimental/ndslice/selection.d

```diff
@@ -0,0 +1,1386 @@
+/**
+$(SCRIPT inhibitQuickIndex = 1;)
+
+Селекторы создают новый тип итератора для тех же данных.
+Сами данные остаются неизменными (to translate).
+
+$(H2 Subspace selectors)
+
+The destination of subspace selectors is painless generalization and combination of other selectors.
+`pack!K` creates a slice of slices `Slice!(N-K, Slice!(K+1, Range))` by packing last `K` dimensions of highest pack of dimensions,
+so type of element of `slice.byElement` is `Slice!(K, Range)`.
+Another way to use `pack` is transposition of packs of dimensions using `evertPack`.
+Examples with subspace selectors are available for selectors, $(SUBREF slice, Slice.shape), $(SUBREF slice, .Slice.elementsCount).
+
+$(BOOKTABLE ,
+
+$(TR $(TH Function Name) $(TH Description))
+$(T2 pack, returns slice of slices.)
+$(T2 unpack, unites all dimension packs.)
+$(T2 evertPack, reverse packs of dimensions.)
+)
+
+$(BOOKTABLE $(H2 Selectors),
+
+$(TR $(TH Function Name) $(TH Description))
+$(T2 blocks, n-dimensional slice of n-dimensional non-overlapping blocks)
+$(T2 windows, n-dimensional slice of n-dimensional overlapping  windows)
```

Comment:
it's not clear here what the difference between blocks and windows are. It would be nice if it was obvious here.